sw/airborne/subsystems/gps.c

```diff
@@ -231,6 +231,8 @@ static void gps_cb(uint8_t sender_id,
   gps = *gps_s;
   AbiSendMsgGPS(GPS_MULTI_ID, now_ts, gps_s);
 #endif
+  gps_time_sync.t0_ticks = sys_time.nb_tick;
```

Comment:
This should not be done on any new message, only when tow was actually updated...